tasks/5.test.js

```diff
@@ -0,0 +1,12 @@
+const { wordSearch } = require("./5");
+
+describe("Task #5", () => {
+  describe("wordSearch()", () => {
+    it("should return true for searching case insensitive word", () => {
+      const result = wordSearch();
```

Comment:
No arguments passed. Also please note that it should be full-word search ;) Meaning: `wordSearch('foobar', 'foo')` should return false

Reply:
Yeah, try the following test cases:
```js
wordSearch('foo', 'foobar') // false
wordSearch('foo', 'Foo bar') // true
wordSearch('foo', ' foo bar') // true
```
Consider looking into word boundary. 😉